src/XeroPHP/Traits/SendEmailTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace XeroPHP\Traits;
+
+use XeroPHP\Remote\Request;
+use XeroPHP\Remote\URL;
+use XeroPHP\Exception;
+
+trait SendEmailTrait
+{
+    public function sendEmail()
+    {
+        /**
+         * @var Object $this
+         */
+        $uri = sprintf('%s/%s/Email', $this::getResourceURI(), $this->getGUID());
+        
+        $url = new URL($this->_application, $uri);
+        $request = new Request($this->_application, $url, Request::METHOD_POST);
+        
+        $request->send();
+        
+        $response = $request->getResponse();
+        
+        if (false !== $element = current($response->getElements())) {
```

Comment:
Ah, is this just copied and pasted from the attachment trait? What does this serve in here?

Reply:
Reckon you could fix it up and I'll merge this in

Reply:
@wobinb can you please remove this line so I can merge this?